### PR TITLE
Hotfix/getlatest

### PR DESCRIPTION
**Pull request by pacmessica**
[notion](https://www.notion.so/figmentnetworks/b47c0f8b935741d8ac28717ddeea2038?v=5e4e1d3854094b788ae2b612329f27e8&p=341c88c4b034429b979092fb75807de0)

disabled metrics because was getting a `nil pointer dereference` error which I don't have time to fix right now, and it's more important to get things syncing again.

```
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: panic: runtime error: invalid memory address or nil pointer dereference
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xf331c2]
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: goroutine 123 [running]:
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: github.com/figment-networks/indexing-engine/metrics.(*GroupCounter).Add(0x0, 0x4014000000000000)
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]:         /home/runner/work/terra-worker/terra-worker/vendor/github.com/figment-networks/indexing-engine/metrics/counter.go:98 +0x22
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: github.com/figment-networks/terra-worker/api.(*Client).SearchTx(0xc000d85040, 0x1495690, 0xc0010d4200, 0x0, 0x0, 0x3c4654, 0x0, 0x0, 0x0, 0x0, ...)
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]:         /home/runner/work/terra-worker/terra-worker/api/transaction.go:43 +0xc7
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: github.com/figment-networks/terra-worker/client.blockAndTx(0x1495690, 0xc0010d4200, 0xc000db4360, 0x148ef30, 0xc000d85040, 0x3c4654, 0x0, 0x0, 0x0, 0x0, ...)
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]:         /home/runner/work/terra-worker/terra-worker/client/client.go:437 +0xca2
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: github.com/figment-networks/terra-worker/client.asyncBlockAndTx(0x1495690, 0xc0010d4200, 0xc000db4360, 0xc000fb3ed0, 0x148ef30, 0xc000d85040, 0xc001145b60)
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]:         /home/runner/work/terra-worker/terra-worker/client/client.go:395 +0x33d
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]: created by github.com/figment-networks/terra-worker/client.getRange
Jul 31 01:02:03 dh--indexer--terra--1 worker[23396]:         /home/runner/work/terra-worker/terra-worker/client/client.go:334 +0x1a6
```

### CODE REVIEW

File: client/pool.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
+package client
```
Comment (lukanus) [inline client/pool.go:1]: wouldn't it be simpler to take what we have in indexer manager and just trigger this one function ?

Comment (pacmessica) [reply client/pool.go:1]: where's that? I just followed the implementation we had in [cosmos](https://github.com/figment-networks/cosmos-worker/blob/dev/client/pool.go) exactly as a quick late friday night fix :p

Comment (lukanus) [reply client/pool.go:1]: I meant this : 
https://github.com/figment-networks/cosmos-worker/pull/31/files#diff-2f645c97a7dc8a38e15d5da6c0ac3ae83933fc85426f3d08a98aed9210b3d102R41

https://github.com/figment-networks/cosmos-worker/pull/31/files#diff-2f645c97a7dc8a38e15d5da6c0ac3ae83933fc85426f3d08a98aed9210b3d102R64
This has corrected, external version of that. 
But if that's what working - go for it

File: api/block.go [hunk 2]
```diff
@@ -4,170 +4,124 @@ import (
...
 	}
 
-	blocks.Lock()
-	defer blocks.Unlock()
-	for _, meta := range result.Result.BlockMetas {
-
-		bTime, _ := time.Parse(time.RFC3339Nano, meta.Header.Time)
-		uHeight, _ := strconv.ParseUint(meta.Header.Height, 10, 64)
-		numTxs, _ := strconv.ParseUint(meta.Header.NumTxs, 10, 64)
+	numTxs := len(result.Result.Block.Data.Txs)
```
Comment (lukanus) [inline api/block.go:83]: Cosmetic: unused so can be just inside structs.Block

**Review by lukanus — APPROVED** [review]

File: client/pool.go (hunk 1 above)
Comment (pacmessica) [reply client/pool.go:1]: I do use that on the new-manager branch: https://github.com/figment-networks/terra-worker/blob/feat/new-manager/client/client.go#L173

So when we finally move colubus-3/4 to k8s, it'll have that

**Pull request merged**